examples/experimental/rke2.yaml

```diff
@@ -0,0 +1,75 @@
+# Deploy kubernetes via rke2 (which installs a bundled containerd).
+# $ limactl start ./rke2.yaml
+# $ limactl shell rke2 sudo kubectl
+#
+# It can be accessed from the host by exporting the kubeconfig file;
+# the ports are already forwarded automatically by lima:
+#
+# $ export KUBECONFIG=$(limactl list rke2 --format 'unix://{{.Dir}}/copied-from-guest/kubeconfig.yaml')
+# $ kubectl get no
+# NAME            STATUS   ROLES                       AGE   VERSION
+# lima-rke2       Ready    control-plane,etcd,master   68s   v1.27.3+rke2r1
+#
+# For more details of RKE2, please refer to https://docs.rke2.io/
+#
+# This example requires Lima v0.7.0 or later.
+
+
+images:
+# Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release-20230518/ubuntu-22.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+  digest: "sha256:afb820a9260217fd4c5c5aacfbca74aa7cd2418e830dc64ca2e0642b94aab161"
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release-20230518/ubuntu-22.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+  digest: "sha256:b47f8be40b5f91c37874817c3324a72cea1982a5fdad031d9b648c9623c3b4e2"
+# Fallback to the latest release image.
+# Hint: run `limactl prune` to invalidate the cache
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+
+# Mounts are disabled in this example, but can be enabled optionally.
+mounts: []
+
+# containerd is managed by rke2, not by Lima, so the values are set to false here.
+containerd:
+  system: false
+  user: false
+
+provision:
+- mode: system
+  script: |
+    #!/bin/sh
+    if [ ! -d /var/lib/rancher/rke2 ]; then
+      curl -sfL https://get.rke2.io | INSTALL_RKE2_CHANNEL=v1.27 sh -
```

Comment:
does the version have to be hardcoded, or is there something like "stable" that could be used instead ?

Reply:
The current stable minor version of RKE2 is `v1.25`, which does not support aarch64.
The hard core here is to consider that keep minor version on `v1.27` until `v1.27` becomes stable. After that the hard core will be removed and the template file will be renamed from `examples/experimental/rke2.yaml` to `examples/rke2.yaml`.
The RKE2 version channels are here [https://github.com/rancher/rke2/blob/master/channels.yaml](https://github.com/rancher/rke2/blob/master/channels.yaml)